app/templates/cases/case/subcases/overview.hbs

```diff
@@ -2,12 +2,13 @@
 
 <Subcases::SubcasesOverviewHeader
   @case={{this.case}}
+  @decisionmakingFlow={{this.decisionmakingFlow}}
```

Comment:
Are both arguments `@case` and `@decisionmakingFlow` required? Feels a bit odd. Can't one of both be skipped and loaded deeper down in the component structure?